haskell_101/codelab/Codelab.hs

```diff
@@ -242,7 +249,7 @@ or :: [Bool] -> Bool
 or l = codelab
 
 
--- "(++)" is the concatenation operator.  To concatenate two linked lists
+-- (++) is the concatenation operator.  To concatenate two linked lists
```

Comment:
Notice that, while not entirely consistent, inline code is in quotes elsewhere.

In Rust the convention is to use backticks in comments when you are referring to code.
I found that style very convenient, as it resolves some ambiguities.
Should we try the same here?
But, I guess, it would better be a separate change.

Reply:
I usually like backticks, the markdown convention, but it can get confusing in Haskell where

    `foo`

is different from

    foo

I've reverted this change; if we want to change how code is formatted, as you say we can do that more coherently in a separate commit.